v1/css.yaml

```diff
@@ -11,20 +11,37 @@ info:
     name: Apache licence, v2
     url: https://www.apache.org/licenses/LICENSE-2.0
 paths:
-  /css/mobile/base:
+  /css/mobile/{file}:
```

Comment:
Bikeshedding... Maybe 'type' or smth?

Reply:
+1, while this does get a file, in the context of the API `type` sounds more appropriate. Let's change that, otherwise LGTM.

Reply:
Ok, I could go with that. The other idea I had would be `bundle`. What do you think of that?

Reply:
> Ok, I could go with that. Another idea I had would be bundle. What do you think of that?

This is a totally bikeshedding session here, but I personally don't like `bundle` - the bundle is what's returned, and the parameter for the bundle is the `type` of the `bundle`.

Reply:
Ok, `type` it is then.

Reply:
Done